sources/problems/steady_state_problem.f90

```diff
@@ -144,6 +140,13 @@ subroutine steady_state_problem_init_base(this)
     class(steady_state_problem_t), intent(inout) :: this
 
     call user_setup(this%C%usr)
```

Comment:
For now, we shouldn't have the user setup. We need to figure out how to support it, but we cannot hi-jack it like this

Reply:
If you check the newest commit, I wouldn't say the problem is 'solved'.
We don't use `user_setup`, so it won't break other examples, but i'm assuming you wouldn't be able to append additional user defined simcomps to this particular example.